container/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# This dockerfile creates a container of a truly basic DIRAC installation which contains only the DIRAC externals.
+# The sole purpose of this container is for developing (and testing) DIRAC "stuff that run",
+# meaning DIRAC components like agents and services. Instructions in ...
+
+# THIS IS A WORK IN PROGRESS
+# TODO:
+#   verify about certificates
+#   correct openssl_config URL location
+#   lcgBundle now fix version
+#   start mySQL
+#   install/start ElasticSearch (?)
+#   install/start ActiveMQ (?)
+
+
+# CC7 as base (the latest tag doesn't work!)
+FROM cern/cc7-base:20170113
+MAINTAINER Federico Stagni <federico.stagni@cern.ch>
+
+# Create DIRAC dirs
+RUN mkdir -p /opt/dirac/DIRAC
```

Comment:
Every line in a dockerfile is a new layer, and each layer eats up space. So you want to factorize your lines.
Here, you can do a single RUN with both directories in the mkdir